internal/client/project.go

```diff
@@ -227,6 +228,14 @@ func WithClientConnect(opts ...serverclient.ConnectOption) Option {
 	}
 }
 
+// WithVars sets variable values from flags and local env on any operations.
+func WithVars(m []*pb.Variable) Option {
```

Comment:
For core you use `WithVariables`. I don't care which it is, but I'd be consistent if you abbreviate variables or not.

Reply:
Set both to `WithVariables`